.vscode/launch.json

```diff
@@ -205,5 +205,18 @@
                 }
             ]
         },
+        {
+            "name": "Python: Remote Attach (3001)",
```

Comment:
Why do we need this 3001 remote attach?

Reply:
I think Sasank added this for testing purposes, perhaps to see if he could attach to two clients running different extensions. I think this change should be removed right?

Reply:
Yep, exactly, feel free to remove it